docs/getting-started.md

```
@@ -0,0 +1,101 @@
+# شروع کار با Cursor AI
+
+## نصب و راه‌اندازی
+
+### سیستم‌های مورد پشتیبانی
+- **Windows**: Windows 10 و بالاتر
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)